src/library/handlers/traceHandler.ts

```diff
@@ -95,42 +101,46 @@ export class TraceHandler {
         }
     }
 
-    public flush(options?: FlushOptions) {
-        // TODO: Flush OpenTelemetry
+    public async flush(): Promise<void> {
+        return this.tracerProvider.forceFlush();
+    }
+
+    public shutdown() {
+        this._spanProcessor.shutdown();
+        this._exporter.shutdown();
     }
 
     // Support Legacy APIs
     public trackRequest(telemetry: Contracts.RequestTelemetry) {
+        let startTime = Date.now();
```

Comment:
Isn't the `startTime` supposed to be before the actual request has been made, not when `trackRequest` is called?

Reply:
Not a blocking comment, but maybe we can let the user pass in the `startTime` as part of the `RequestTelemetry`. This way, the `startTime` could potentially represent when the actual start time of the request was made.